pycqed/instrument_drivers/meta_instrument/qubit_objects/QuDev_transmon.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import numpy as np
+
+from qcodes.instrument.parameter import ManualParameter
+
+from pycqed.measurement import detector_functions as det
+from pycqed.measurement import mc_parameter_wrapper as pw
+from pycqed.analysis import measurement_analysis as ma
+
+from pycqed.instrument_drivers.meta_instrument.qubit_objects.qubit_object \
+    import Qubit
+
+
+class QuDev_transmon(Qubit):
+    def __init__(self, name, MC, heterodyne_instr, cw_source, **kw):
+        super().__init__(name, **kw)
+
+        self.MC = MC
+        self.heterodyne_instr = heterodyne_instr
+        self.cw_source = cw_source
+
+        self.add_parameter('f_RO_resonator', label='RO resonator frequency',
+                           units='Hz', initial_value=0,
```

Comment:
Thanks for the tip. I updated to the latest QCoDeS and changed `units` to `unit` in all code I wrote/changed.